R/cppproperties/extractLinkingTo.R

```diff
@@ -0,0 +1,8 @@
+
+deps <- unname(read.dcf("DESCRIPTION", "LinkingTo")[1, ]) # Read 'LinkingTo' field from description
+deps <- gsub("\\s|\\n|(\\([^\\)]*\\))", "", deps) # Remove all whitespace and line breaks
+deps <- strsplit(deps, ",")[[1]] # Split package names
```

Comment:
Just for reference, [`desc:::parse_deps()`](https://github.com/r-lib/desc/blob/1b97c360d96f0fe22d225919e5f20ac2a08b870f/R/deps.R#L95-L108) does almost the same thing. Not sure if there is any edge case we don't cover?

Reply:
I had a look at this function when writing this code, it's more complex because they also parse version constraints (inside brackets) while we don't need to so they are deleted by the regex. I'll add this info to the comment for clarification.